postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
@blbradley, agreed. My fault, sorry.

The most reliable way to test Salt modules is to write tests with `unittest`, but sometimes it's tedious to write them, so it's easier test Salt by hands. But I want exhaustive testing, not just in one "special" OS.
How do you test modules? Are there Docker/Vagrant/etc. containers for testing Salt states in automated way?